ami/interactions/environments/sensors/soundcard_audio_sensor.py

```diff
@@ -0,0 +1,123 @@
+import math
+import threading
+import time
+from collections import deque
+
+import torch
+from torch import Tensor
+from typing_extensions import override
+from vrchat_io.audio import SoundcardAudioCapture
+
+from .base_sensor import BaseSensor
+
+
+class SoundcardAudioSensor(BaseSensor[Tensor]):
+    @override
+    def __init__(
+        self,
+        device_name: str | None = None,
+        sample_rate: float = 16000,  # 16 kHz
+        channel_size: int = 2,
+        read_sample_size: int = 1600,  # 0.1 sec
```

Comment:
`read_size`か`read_samples`どっちかだけでも伝わりそう (ごめん前のコメントで変数名の候補提案しとけばよかった)
```suggestion
        read_size: int = 1600,  # 0.1 sec
```